pkg/hooks/hooks.go

```diff
@@ -56,7 +56,8 @@ type Hooks struct {
 	AddMultinodeStatsHook      func(indexData utils.AllIndexesStats, orgId uint64,
 		logsIncomingBytes *float64, logsOnDiskBytes *float64, logsEventCount *int64,
 		metricsIncomingBytes *uint64, metricsOnDiskBytes *uint64, metricsDatapointsCount *uint64,
-		queryCount *uint64, totalResponseTime *float64)
+		queryCount *uint64, totalResponseTimeSinceRestart *float64, totalResponseTimeSinceInstall *float64,
+		totalQueryCountSinceInstall *uint64, totalColumnsSet map[string]struct{})
```

Comment:
nit: this is a lot of parameters. It would probably be good to make some struct types to store some of these that are semantically similar. But I don't think it's necessary to do that in this PR

Reply:
Yes for now I will add this as a TODO.